src/modules/settings/pages/about.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
it looks like this page file is named differently.

Reply:
It was originally lowercase and now uppercase, but git won't pick up it changed